packages/@aws-cdk/aws-codebuild/test/linux-arm-build-image.test.ts

```diff
@@ -0,0 +1,331 @@
+import { Match, Template } from '@aws-cdk/assertions';
+import * as ecr from '@aws-cdk/aws-ecr';
+import * as cdk from '@aws-cdk/core';
+import { BuildSpec } from '../lib/build-spec';
+import { PipelineProject } from '../lib/pipeline-project';
+import { ComputeType, Project } from '../lib/project';
+import { LinuxArmBuildImage } from '../lib/linux-arm-build-image';
```

Comment:
Can you substitute these with the qualified import? (`import * as codebuild from '../lib'`)?

If not, and you end up with a cyclic dependency error, I would be worried about releasing this in this state (there's a big chance CDK customers would get the same error when trying to use the new `LinuxArmBuildImage` class).

Reply:
Yes, shouldn't be a problem as most other tests use the qualified imports and stopped erroring out. Updated commit coming in a few.